Add homepage to remix app

alse opened the pull request:
#153

- Adds homepage to remix app
- Misc style updates

10thfloor commented:
Nice! 

A few things I noticed:


1) Are these placeholders? The quality is bad. :(
<img width="251" alt="Screen Shot 2022-06-14 at 4 06 36 PM" src="https://user-images.githubusercontent.com/901466/173703725-da1498a4-cc0a-4d0b-97ca-a706a284c0b2.png">

2) The icons are clipping here.
We can apply `width` `height` and `viewBox` to the imported svgs if necessary.
<img width="241" alt="Screen Shot 2022-06-14 at 4 06 43 PM" src="https://user-images.githubusercontent.com/901466/173703789-0d82b77c-e6eb-486b-be8a-9ae6bc5e9754.png">

3) Dates are not formatted here: use `date-fns`
<img width="145" alt="Screen Shot 2022-06-14 at 4 07 16 PM" src="https://user-images.githubusercontent.com/901466/173703889-fc79cf8b-afa9-41a4-8948-560e0bc39fb9.png">

4) There is a slight shadow on hover on the top "Card" but not the bottom one. They should both have a hover state?
<img width="1330" alt="Screen Shot 2022-06-14 at 4 03 07 PM" src="https://user-images.githubusercontent.com/901466/173703952-2b6d0ca3-1c78-4dfc-b5ef-d7a454bfedca.png">

 5) Clicking on these tabs causes a 500 error. Not sure but i think this will require a deeper discussion of how we want these tabs to work with Remix routes / links
<img width="1554" alt="Screen Shot 2022-06-14 at 4 02 51 PM" src="https://user-images.githubusercontent.com/901466/173703999-564ea461-19d0-47fa-a349-a672f85424e2.png">
 
That;s it for now, otherwise amazing to see this live!

10thfloor reviewed (changes requested):
See my comments in the main PR thread.

alse commented:
Thanks for the review @10thfloor , let me know if this needs any other changes!

1. Yes, those are placeholders that we'll have to update before launch
2. Good catch, it looks like some of the svgs need to be updated after the svgr changes. I fixed these and will keep an eye on any others
3. I added a helper function to format the dates, and we'll have to use the same method for all other dates in a future PR
4. I removed the shadow on the bottom one because only the 2 internal links are clickable, not the entire card. Should we add the hover state anyway, or remove it from both?
5. Yes, we'll have to figure out how we want to load the tabs. We could use links/routes, load them client side, or preload everything in the page loader
